shared/attributes.asciidoc

```diff
@@ -18,25 +18,34 @@
 :winlogbeat-ref:       https://www.elastic.co/guide/en/beats/winlogbeat/{branch}
 :heartbeat-ref:        https://www.elastic.co/guide/en/beats/heartbeat/{branch}
 :journalbeat-ref:      https://www.elastic.co/guide/en/beats/journalbeat/{branch}
-:apm-get-started-ref:  https://www.elastic.co/guide/en/apm/get-started/{branch}
 :apm-overview-ref-v:   https://www.elastic.co/guide/en/apm/get-started/{branch}
 :apm-overview-ref-70:  https://www.elastic.co/guide/en/apm/get-started/7.0
-:apm-server-ref:       https://www.elastic.co/guide/en/apm/server/{branch}
 :apm-server-ref-v:     https://www.elastic.co/guide/en/apm/server/{branch}
-:apm-server-ref-62:    https://www.elastic.co/guide/en/apm/server/6.2
-:apm-server-ref-64:    https://www.elastic.co/guide/en/apm/server/6.4
 :apm-server-ref-70:    https://www.elastic.co/guide/en/apm/server/7.0
+:apm-server-ref-64:    https://www.elastic.co/guide/en/apm/server/6.4
+:apm-server-ref-62:    https://www.elastic.co/guide/en/apm/server/6.2
 :apm-agents-ref:       https://www.elastic.co/guide/en/apm/agent
+:apm-py-ref-v:         https://www.elastic.co/guide/en/apm/agent/python/{apm-py-branch}
```

Comment:
It looks like we've been using `-ref` without the `-v` part. Do you think we can stick to that or is it broken for you?

Reply:
Good question. `-v` is what we've used for the past 12 months to version links from the APM Server to the APM Agents. Here:

https://github.com/elastic/apm-server/blob/862035f4edaeed0e9c6dc41eed4efbb2006f9a32/docs/version.asciidoc#L27-L33

The benefit of moving this from APM Server -> docs is that these `-v` attributes can now be used across the stack.

After this PR is merged I'll go through and update all `-ref` links to `-v` links, then remove the `-ref` attributes entirely.

Reply:
My idea was to put the `ref` links in and "outdated APM" section of the attributes file so others wouldn't use them. This would give me time to update the `-ref` links into `-v` links and remove them.

Reply:
I like the plan but wonder if we could use the `-ref` links instead of `-v` links.

Reply:
I don't see why not. If you think that's the preferred naming convention, I'm happy to update this PR.

Reply:
Just changes which links I need to update. I don't see that being a problem.

Reply:
It's *a lot* more links to update but with find and replace it doesn't really matter.

Reply:
I *think* it is preferred. @lcawl, do you agree?

Reply:
I think omitting the -v makes them align better with the other URL attributes, but ultimately it's up to you @bmorelli25 since I think you'll be using them the most.